apps/codeforafrica/src/payload/globals/Footer.js

```diff
@@ -0,0 +1,168 @@
+import { array } from "payload/dist/fields/validations";
+
+import linkArray from "../fields/links/linkArray";
+
+const Footer = {
+  slug: "footer",
+  access: {
+    read: () => true,
+  },
+  label: "Footer",
+  fields: [
+    {
+      name: "logo",
+      type: "upload",
+      relationTo: "media",
+      required: true,
+      filterOptions: {
+        mimeType: { contains: "image" },
+      },
+      label: "Logo",
+    },
+    {
+      name: "description",
+      type: "code",
```

Comment:
This needs to be `richText`, no?

Reply:
@kilemensi from this [link](https://dev.codeforafrica.org), I see the description has some links in it. That informed the decision of going with code instead of richText here

Reply:
Nah, [`code`](https://payloadcms.com/docs/fields/code) is for actual code i.e. Python, JavaScript, HTML, etc. @koechkevin. [`richText`](https://payloadcms.com/docs/fields/rich-text) is for well, rich text content. Content that can include formatting (bold, italic, etc.), link, images, etc.

We need rich text.